src/averageAges.js

```diff
@@ -15,6 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const arrayOfMen = people.filter(person => {
+    return arguments.length < 2
+      ? person.sex === 'm'
+      : Math.ceil(person.died / 100) === century && person.sex === 'm';
+  });
+
+  const sumOfAges = arrayOfMen.reduce((acc, man) => {
+    return acc + (man.died - man.born);
+  }, 0);
```

Comment:
move this reduce into separated function and reuse throughout your code. Because you have duplicated this code in a few places